msm/skill_state.py

```diff
@@ -22,6 +23,10 @@ def load_device_skill_state() -> dict:
 
 
 def write_device_skill_state(data: dict):
+    try:
+        os.system("mkdir ~/.mycroft")
```

Comment:
I suggest you get the directory from the SKILL_STATE_PATH global,

```python
dir_path = dirname(expanduser(SKILL_STATE_PATH))
```

For the creating maybe pathlib's mkdir or os.mkdirs could be used?